### ISSUE TITLE
merge.py", line 18, in <module>     theta_0 = model_0["state_dict"] KeyError: 'state_dict'

### ISSUE THREAD

**Issue by clkky1** [eyriewow/merge-models#3]
merge.py", line 18, in <module>
theta_0 = model_0["state_dict"]
KeyError: 'state_dict'

Excuse me, I'm a beginner maybe my question is a bit stupid and I didn't find a valid solution so I'm asking. When I try to use Merge Models I installed all the environment I need, my system is windows10, but I follow the operation and after running the .bat file for a while it prompts me "keyError". Can you provide me with some effective advice？

**Comment by clkky1** [comment]
Already solved ^_^

**Comment by eyriewow** [comment]
Hey, 
I was unfortunately a bit busy and forgot to check in - I'm happy you figured it out yourself.   
Could you please tell me what the issue ended up being and how you solved it? I'd like to be able to reference this if it comes up in the future.

**Comment by clkky1** [comment]
Sorry I've been preparing for my discrete math exam with no response. Actually I didn't solve this problem and I looked for an alternative. 
Because I have been obsessed with using the (animefull-latest 7.17G) model as the main model for weight fusion, but the prompt - (KeyError: 'state_dict') is caused by the animefull-latest model. I don't know why. 
After that I tested other models and found that they worked fine for weight fusion. So I replaced the (animefull-final-pruned 4.16G+animesfw-final-pruned4.16G) fused model as my main model and now everything is fine
